### PR TITLE
Start all script

### PR DESCRIPTION
**Pull request by web-devel**
* use Postgres in `scr`;
* add `start-react-all` script;

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,10 @@
     "build-generator": "lerna run --scope @cuba-platform/front-generator build",
     "gen-client-scr": "npm run build-generator && lerna run --scope @cuba-platform/front-generator gen-client-scr",
     "update-client-libs": "node scripts/update-client-libs.js",
-    "bootstrap-client-app": "npm run build-client-libs && npm run pack-client-libs && npm run gen-client-scr && npm run update-client-libs"
+    "bootstrap-react-app": "npm run build-client-libs && npm run pack-client-libs && npm run gen-client-scr && npm run update-client-libs",
+    "start-cuba-app": "cd ./sample-car-rent && ./gradlew setupTomcat createDb deploy start",
```
Comment (vadimbasko) [inline package.json]: I suggest rename start-cuba-app -> bootstrap-cuba-app because we create database here.

File: package.json [hunk 2]
```diff
@@ -10,7 +10,10 @@
...
+    "start-react-app": "cd ./react-client-scr && npm run start",
+    "start-react-all": "npm run bootstrap-react-app && npm run start-cuba-app && npm run start-react-app"
```
Comment (vadimbasko) [inline package.json]: May be better name is "bootstrap-and-run-all", so we need to distinguish generation and run actions.

Also, I'm not sure about 'bootstrap' word at all, may be it make sense to find shorter synonym, like 'init'.

Comment (vyacheslav-pushkin) [reply package.json]: `init` sounds good to me, perhaps we could replace `start` with `run` as well.

Also we need to differentiate between clients, so I would suggest to keep `react` inside script name. I was thinking about something along the lines of `run-react` / `run-cuba` / `init-run-react-all`.

Comment (web-devel) [reply package.json]: `run-` looks strange when using with npm: `npm run run-react`
+1 for distinguishing clients that is why it's contains `-react` suffix.

**Pull request merged**